p8e-api/src/main/kotlin/io/provenance/engine/domain/Transaction.kt

```diff
@@ -61,7 +61,7 @@ class BlockResultsRequest(
 data class BlockResults(
     val height: Long,
     @JsonProperty("txs_results")
-    val txsResults: List<TxResult>
+    val txsResults: List<TxResult>?
```

Comment:
not sure if the spec changed so the endpoint that returns this can return null now instead of an empty array, but I was getting null locally and had to fix